### PR TITLE
[#135504871]Add_New_Customer

### PR DESCRIPTION
**Pull request by tanjunlem**

### CODE REVIEW

File: process/code/AddCustomer_Coding.txt [hunk 1]
```diff
@@ -0,0 +1,150 @@
+public class AddCustomer extends javax.swing.JDialog {
+
+    public AddCustomer() {
+        initComponents();
+        setLocationRelativeTo(this);
+    }
+
+    @SuppressWarnings("unchecked")
+    // <editor-fold defaultstate="collapsed" desc="Generated Code">
```
Comment (TeckYong) [inline process/code/AddCustomer_Coding.txt:9]: what does this code mean ?

Comment (tanjunlem) [reply process/code/AddCustomer_Coding.txt:9]: This code is the function or event let the staff to add the new customer into the system.

Comment (TeckYong) [reply process/code/AddCustomer_Coding.txt:9]: Oic. Thanks

**Review by TeckYong — APPROVED** [review]

File: process/code/AddCustomer_Coding.txt (hunk 1 above)
Comment (tanjunlem) [reply process/code/AddCustomer_Coding.txt:9]: You are welcome

**Comment by MasterODST** [comment]
Nothing to say but good work you did here.

**Pull request merged**